src/core/js/views/adaptView.js

```diff
@@ -171,6 +172,10 @@ define([
                 return this.$el.removeClass('display-none');
             }
             this.$el.addClass('display-none');
+        },
+        
+        addCompletionClass:function(){
+            this.$el.addClass('completed');
```

Comment:
needs to take account of the value change. it may be changing from true to false.
```js
onChangeComplete: function(value) {
    this.$el[value?'addClass':'removeClass']('completed');
}
```
```js
onChangeComplete: function(value) {
    if (!value) {
        this.$el.removeClass('completed');
    } else {
        this.$el.addClass('completed');
    }
}
```

Reply:
```js
onIsCompleteChange: function(isComplete) {
    this.$el.toggleClass('completed', isComplete);
}
```

Reply:
Nice that toggle takes the second argument. Will have to remember that.